### PR TITLE
Fix freq new mne

### LINKED ISSUE #111: ImportError: cannot import name 'psd_welch' from 'mne.time_frequency'
New mne does not have `psd_welch`. I'll have to see if new mne Spectrum classes can replace `borsar.freq.PSD`.

### PR DESCRIPTION
**Pull request by mmagnuski**
Hopefully fixes #111.

### CODE REVIEW

File: borsar/freq.py [hunk 1]
```diff
@@ -188,9 +197,18 @@ def compute_psd(inst, tmin=None, tmax=None, winlen=None, step=None, padto=None,
...
         n_per_seg, n_overlap, n_fft = _psd_welch_input_seconds_to_samples(
             inst, winlen, step, padto)
-        psd, freq = psd_welch(inst, tmin=tmin, tmax=tmax, n_fft=n_fft,
-                              picks=picks, n_per_seg=n_per_seg,
-                              n_overlap=n_overlap)
+
+        if new_psd:
+            args = dict(n_fft=n_fft, n_overlap=n_overlap, n_per_seg=n_per_seg)
+            spectrum = inst.compute_psd(method='welch', tmin=tmin, tmax=tmax,
+                                        picks=picks, method_kw=args)
```
Comment (drammock) [inline borsar/freq.py:204]: `method_kw` is a `**` argument; you can either do this:
```suggestion
                                        picks=picks, **args)
```

Or you can just include `n_fft=n_fft, n_overlap=n_overlap, n_per_seg=n_per_seg` directly in the `compute_psd` call here.

File: borsar/freq.py [hunk 2]
```diff
@@ -188,9 +197,18 @@ def compute_psd(inst, tmin=None, tmax=None, winlen=None, step=None, padto=None,
...
+            freq = spectrum.freqs
+            psd = spectrum.get_data()
```
Comment (drammock) [inline borsar/freq.py:206]:
```suggestion
            psd, freq = spectrum.get_data(return_freqs=True)
```

**Comment by mmagnuski** [comment]
Thanks for your suggestions @drammock!

File: borsar/freq.py (hunk 1 above)
Comment (mmagnuski) [reply borsar/freq.py]: Thanks! I was too lazy to test this locally ;)

**Comment by mmagnuski** [comment]
Unfortunatelly I pushed some changes before I saw your comments and now, I can't use the "commit suggestion" button so you could be the author of the commit changes.

File: borsar/freq.py (hunk 2 above)
Comment (mmagnuski) [reply borsar/freq.py]: oh, good to know!

**Comment by drammock** [comment]
> Unfortunatelly I pushed some changes before I saw your comments and now, I can't use the "commit suggestion" button so you could be the author of the commit changes.

it doesn't really matter to me. If it bothers you though, you can add this to the commit message (on its own line) when you merge:

```
Co-authored-by: Daniel McCloy <dan@mccloy.info>
```

see [here](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors#creating-co-authored-commits-on-the-command-line)

**Pull request merged**